README.md

```diff
@@ -1,4 +1,105 @@
+* `agreement`: An array that can contain one or more specific agreement clauses, such as those related to payments or permissions. Each element in this array specifies a type of agreement and its details.
+* `extensions`: A section for adding any custom or non-standard fields relevant to a specific use case.
+
+### Use Case: Permissions
+
+When the type within the agreement array is set to "permission", the structure defines the access rights granted to a grantee for a specific asset. Relevant fields within this agreement type (as seen in [`sacd.permission.example.json`](data/sacd.permission.example.json)) include:
+
+* `asset`: A DID identifying the asset to which permissions are being granted, such as a specific NFT (e.g., did:nft:137:0x3330000000000000000000000000000000000000_31415).
+* `permissions`: An array detailing the specific permissions being granted. Each permission typically includes a name (e.g., "commands", "location:View:Current") and a human-readable description.
+* `attachments`: An array of documents related to the permission agreement, each with a name, description, contentType, and uri (e.g., for a legal agreement).
+* `signatures`: An array of signatures from the involved parties, including the signer's address, the signature itself, and a timestamp.
+
+### Use Case: Payments
+
+When the type within the agreement array is set to "payment", the structure defines the terms of a financial transaction. Relevant fields within this agreement type (as seen in [`sacd.payment.example.json`](sata/sacd.payment.example.json)) include:
+
+* `asset`: A DID identifying the payment asset, such as a specific ERC-20 token (e.g., did:erc20:137:0x4440000000000000000000000000000000000000).
+* `payment`: An object containing the payment details:
+    * `amount`: The amount to be paid (e.g., "125000000000000000000000").
+    * `recurrence`: The frequency of payment (e.g., "monthly", "one-time").
+    * `terms`: Additional payment terms, which can include:
+        * `initialPayment`: The initial payment amount.
+        * `paymentMethod`: The method of payment (e.g., "direct transfer").
+* `purpose`: A description of what the payment is for (e.g., "Subscription service for vehicle data").
+* `attachments`: An array of documents related to the payment agreement, each with a name, description, contentType, and uri (e.g., for a service agreement or payment schedule).
+* `signatures`: An array of signatures from the involved parties, including the signer's address, the signature itself, and a timestamp.
+
+This documentation should provide a foundational understanding of the SACD JSON format and its usage for defining payment and permission agreements within the DIMO ecosystem. More complex agreements may include additional fields within the agreement section or utilize the extensions field for specific requirements.
+
+### NFT DID Format
+
+The DID (Decentralized Identifier) follows the format:
```

Comment:
I think this is not the only sort of DID we'll have, I would make that clear. We already use `erc20` above!

Reply:
We can also link to the Ceramic spec here.